tests/test_rbd_csi_default_sc.py

```diff
@@ -0,0 +1,208 @@
+    config.load_kube_config()
+    v1 = client.CoreV1Api()
+    ret = v1.list_pod_for_all_namespaces(
+        watch=False,
+        label_selector='app=rook-ceph-tools'
+    )
+
+    for i in ret.items:
+        namespace = i.metadata.namespace
+        name = i.metadata.name
+        break
+    tool_box = pod.Pod(name, namespace)
+    return tool_box
+
+def validate_storageclass(sc_name):
+    """
+    Validate if storageClass is been created or not
+    """
+    sc_obj = SC.get(resource_name="")
+    sample = sc_obj['items']
+    for item in sample:
+        if item.metadata.name in sc_name:
+            log.info(f"StorageClass got created successfully")
+            return True
+    return False
+
+def create_secret_rbd(**kwargs):
+    """
+    This will create Secret file which will be used for creating StorageClass
+
+    :return:
+    """
+    file_y = templating.generate_yaml_from_jinja2_template_with_data(
+        SECRET_RBD_YAML, **kwargs
+    )
+    with open(TEMP_YAML_FILE, 'w') as yaml_file:
+        yaml.dump(file_y, yaml_file, default_flow_style=False)
+    assert SECRET.create(yaml_file=TEMP_YAML_FILE)
+    return True
+
+def get_client_admin_keyring():
+    """
+    This will fetch client admin keyring from Ceph
+
+    :return:
+    """
+    cmd = f"ceph auth get-key client.admin|base64"
+    tool_box = get_toolbox_pod_handler()
+    out, err, ret = tool_box.exec_command(cmd=cmd, timeout=20)
+    if out:
+        out = out.rstrip('\n')
+        secret_data = {}
+        secret_data['client_admin_key'] = out
+        assert create_secret_rbd(**secret_data)
+    else:
+        log.error(f"Failed to get client auth key")
+        return StatusOfTest.FAILED
+
+def create_pvc():
+    """
+    This will create PVC with default value
+
+    :return:
+    """
+
+    file_y = templating.generate_yaml_from_jinja2_template_with_data(
+        PVC_RBD_YAML,
+    )
+    with open(TEMP_YAML_FILE, 'w') as yaml_file:
+        yaml.dump(file_y, yaml_file, default_flow_style=False)
+    assert PVC.create(yaml_file=TEMP_YAML_FILE)
+    log.info(f"Waiting for 3 sec for PVC to get created")
+    sleep(3)
+    return True
+
+def validate_pvc(pvc_name):
+    """
+    This will check if PVC created is bound successfully or not
+
+    :param pvc_name:
+    :return:
+    """
+    pvc_list = PVC.get(resource_name=pvc_name)
+    pvc_status=pvc_list['status'].phase
+    if pvc_status == "Bound":
+        log.info(f"PVC is created and Status is {pvc_status}")
+        return True
+    else:
+        log.info(f"Failed: PVC status PVC in {pvc_status}")
+        return False
+
+@tier1
+class TestCaseOCS347(EcosystemTest):
+    def test_347(self):
+
+        assert create_rbd_pool()
+        assert validate_pool_creation("rbd")
+        assert create_storageclass_rbd()
+        assert validate_storageclass("ocsci-csi-rbd")
+        get_client_admin_keyring()
+        assert create_pvc()
+        assert validate_pvc("rbd-pvc")
+        return StatusOfTest.PASSED
```

Comment:
Add empty line at the end of the file.
pep8 requires an empty line

Reply:
Please ensure, that each file is ending with new line (also `pool.yaml` and `storageclass-csi-rbd.yaml`).